src/GitLinkTask/GitLink.cs

```diff
@@ -18,21 +18,33 @@ public class GitLink : Task
         [Required]
         public ITaskItem PdbFile { get; set; }
 
-        public bool DownloadWithPowershell { get; set; }
+        public string Method
+        {
+            get { return this.MethodEnum.ToString(); }
```

Comment:
Is it the property *getter* specifically, that you want to get rid of? Because the property setter at least is important. And a setter without a getter is a little odd. Especially when we can support both.